.github/workflows/deploy-web.yml

```
@@ -99,11 +99,26 @@ jobs:
       - name: Success Code
         run: exit 0
 
+  get-branch-or-tag-sha:
+    name: Get Branch or Tag SHA
+    runs-on: ubuntu-22.04
+    outputs:
+      branch-or-tag-sha: ${{ steps.get-branch-or-tag-sha.outputs.sha }}
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
```

❓ Would this need to target the specified branch that a user chooses as the input for `branch-or-tag`?

```suggestion
    steps:
      - name: Checkout Branch
        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
        with:
          ref: ${{ inputs.branch-or-tag }}

```

---

I thought about that, but since I'm using `ls-remote` to get SHA, it doesn't need to target the branch or tag, but this is close to home. I will use this ad use a simple command to get the SHA